<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {background-color: rgb(0, 255, 255);
font: 14pt &quot;Arial&quot;;
border-radius: 10px;
border-width: 2px;
hover: rgb(212, 255, 252)
}

QPushButton:hover {
background-color: rgb(100, 255, 255);
}

QPushButton:pressed {
background-color: rgb(27, 198, 186)
}

QLabel{
font: 14pt &quot;Arial&quot;;
}

#logout_button {background-color: rgb(255, 80, 80);
font: 14pt &quot;Arial&quot;;
border-radius: 10px;
border-width: 2px;
hover: rgb(212, 255, 252)
}

#logout_button:hover {
background-color: rgb(255, 136, 136);
}

#logout_button:pressed {
background-color: rgb(186, 3, 3)
}

#welcome_label_analista_calidad{
font: 18pt &quot;Arial&quot;;
}
</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../media/imagenes/logo.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="titleDash">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>129</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RESULTADOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SALIR</string>
   </property>
  </widget>
  <widget class="QTableView" name="usuariosPromedios">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>451</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTableView" name="erroresOperativos">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="desviacionesGraves">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>320</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="promedioGeneral">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>PROMEDIO GENERAL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalEvaluaciones">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TOTAL EVALUACIONES: </string>
   </property>
  </widget>
  <widget class="QLabel" name="totalCeros">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TOTAL CEROS: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="filtroProceso">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="erroresOpsLABEL">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ERRORES OPERATIVOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="desviacionesGravesLabel">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>DESVIACIONES GRAVES</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
